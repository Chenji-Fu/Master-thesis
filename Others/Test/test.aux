\relax 
\providecommand\hyper@newdestlabel[2]{}
\providecommand\HyperFirstAtBeginDocument{\AtBeginDocument}
\HyperFirstAtBeginDocument{\ifx\hyper@anchor\@undefined
\global\let\oldcontentsline\contentsline
\gdef\contentsline#1#2#3#4{\oldcontentsline{#1}{#2}{#3}}
\global\let\oldnewlabel\newlabel
\gdef\newlabel#1#2{\newlabelxx{#1}#2}
\gdef\newlabelxx#1#2#3#4#5#6{\oldnewlabel{#1}{{#2}{#3}}}
\AtEndDocument{\ifx\hyper@anchor\@undefined
\let\contentsline\oldcontentsline
\let\newlabel\oldnewlabel
\fi}
\fi}
\global\let\hyper@last\relax 
\gdef\HyperFirstAtBeginDocument#1{#1}
\providecommand\HyField@AuxAddToFields[1]{}
\providecommand\HyField@AuxAddToCoFields[2]{}
\@writefile{toc}{\contentsline {section}{\numberline {1}Introduction}{1}{section.1}\protected@file@percent }
\@writefile{toc}{\contentsline {section}{\numberline {2}Description of the connected component $X$ containing a TRSELP}{2}{section.2}\protected@file@percent }
\@writefile{toc}{\contentsline {subsection}{\numberline {2.1}Tame regular semisimple elliptic $L$-parameters}{2}{subsection.2.1}\protected@file@percent }
\@writefile{toc}{\contentsline {subsection}{\numberline {2.2}Description of the component}{3}{subsection.2.2}\protected@file@percent }
\newlabel{Theorem: X}{{1}{4}{}{theorem.1}{}}
\@writefile{toc}{\contentsline {subsection}{\numberline {2.3}the $T$-action on $(\hat  {G} \times T \times \mu )$}{5}{subsection.2.3}\protected@file@percent }
\@writefile{toc}{\contentsline {section}{\numberline {3}Main Theorem: description of $[X/\hat  {G}]$}{6}{section.3}\protected@file@percent }
\@writefile{toc}{\contentsline {section}{\numberline {4}Corollary: lifting irreducible coherent sheaves on $[X/\hat  {G}]$}{9}{section.4}\protected@file@percent }
\@writefile{toc}{\contentsline {section}{\numberline {5}Example}{9}{section.5}\protected@file@percent }
\bibcite{Con14}{Con14}
\bibcite{dat.pdf}{dat.pdf}
\bibcite{DHKM20}{DHKM20}
\bibcite{DR09}{DR09}
\bibcite{Edixhoven}{Edixhoven}
\bibcite{FS}{FS}
\bibcite{Kal19}{Kal19}
\bibcite{Vig97}{Vig97}
