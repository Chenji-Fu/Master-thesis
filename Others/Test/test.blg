This is BibTeX, Version 0.99d (TeX Live 2020/W32TeX)
Capacity: max_strings=200000, hash_size=200000, hash_prime=170003
The top-level auxiliary file: test.aux
The style file: alpha2.bst
Database file #1: test_ref.bib
You've used 3 entries,
            2549 wiz_defined-function locations,
            577 strings with 4790 characters,
and the built_in function-call counts, 1014 in all, are:
= -- 93
> -- 41
< -- 2
+ -- 14
- -- 13
* -- 65
:= -- 174
add.period$ -- 8
call.type$ -- 3
change.case$ -- 16
chr.to.int$ -- 3
cite$ -- 3
duplicate$ -- 41
empty$ -- 84
format.name$ -- 18
if$ -- 203
int.to.chr$ -- 1
int.to.str$ -- 0
missing$ -- 3
newline$ -- 17
num.names$ -- 9
pop$ -- 23
preamble$ -- 1
purify$ -- 20
quote$ -- 0
skip$ -- 34
stack$ -- 0
substring$ -- 47
swap$ -- 3
text.length$ -- 2
text.prefix$ -- 2
top$ -- 0
type$ -- 20
warning$ -- 0
while$ -- 9
width$ -- 5
write$ -- 37
