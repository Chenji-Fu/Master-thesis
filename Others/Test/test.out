\BOOKMARK [1][-]{section.1}{Introduction}{}% 1
\BOOKMARK [1][-]{section.2}{Description of the connected component X containing a TRSELP}{}% 2
\BOOKMARK [2][-]{subsection.2.1}{Tame regular semisimple elliptic L-parameters}{section.2}% 3
\BOOKMARK [2][-]{subsection.2.2}{Description of the component}{section.2}% 4
\BOOKMARK [2][-]{subsection.2.3}{the T-action on \( T \)}{section.2}% 5
\BOOKMARK [1][-]{section.3}{Main Theorem: description of [X/]}{}% 6
\BOOKMARK [1][-]{section.4}{Corollary: lifting irreducible coherent sheaves on [X/]}{}% 7
\BOOKMARK [1][-]{section.5}{Example}{}% 8
